requirements_test_all.txt

```diff
@@ -173,6 +173,9 @@ elgato==0.2.0
 # homeassistant.components.emulated_roku
 emulated_roku==0.1.9
 
+# homeassistant.components.entur_public_transport
```

Comment:
Is this intentionial in this PR?

Reply:
No, I don't actually know why that is included.

Reply:
It happens 😁

Reply:
I think something in the dev branch is wrong with that particular dependency as I can't run the requirements generator script without bringing that entur depencency back.